extract_orgchart_data/extract_orgchart_data.py

```diff
@@ -0,0 +1,24 @@
+from helpers.extract_ministers_departments import extract_ministers_departments
+from helpers.write_to_csv import write_to_csv
+from helpers.crawl_pdfs import download_all_pdfs
+from helpers.get_pdf_names import get_pdf_names
+import os
+
+WEBSITE_URL = "http://www.cabinetoffice.gov.lk/cab/index.php?option=com_content&view=article&id=54&Itemid=93&lang=en"
+PDF_DIRECTORY = "pdfs"
+CSV_DIRECTORY = "extracted"
+
+if __name__ == "__main__":
+
+    download_all_pdfs(WEBSITE_URL, PDF_DIRECTORY)
+    pdf_file_names = get_pdf_names(PDF_DIRECTORY)
+
+    for pdf_file_name in pdf_file_names:
+        # extract ministers and corresponding departments
+        pdf_location = os.path.join(os.getcwd(), PDF_DIRECTORY, pdf_file_name)
+        extracted_data = extract_ministers_departments(pdf_location)
+
+        # writing to csv
+        write_to_csv(extracted_data,pdf_file_name,CSV_DIRECTORY)
```

Comment:
There are some linting issues. Try to check in other places. You can install Python linters in your IDE and check for these.
```suggestion
        write_to_csv(extracted_data, pdf_file_name, CSV_DIRECTORY)
```